pages/index.tsx

```diff
@@ -1,9 +1,80 @@
+import '@rainbow-me/rainbowkit/styles.css'
 import type { NextPage } from 'next'
 
+import { ConnectButton } from '@rainbow-me/rainbowkit'
+import { useAccount, useSignMessage } from 'wagmi'
+import { useCallback, useEffect, useState } from 'react'
+import { getSupabase } from '../utils/supabase'
+
+type User = {
+  name: string
+}
+
 const Home: NextPage = () => {
+  const [message, setMessage] = useState()
+  const [user, setUser] = useState<User>()
+  const [token, setToken] = useState('')
```

Comment:
tokens should be secret/obfuscated, right?
Would an experienced developer be able to see this token and use it elsewhere?

Reply:
Yeah, a hacker could easily get them. We will need to store them in cookies for now.

Reply:
I'll create an issue for this. For now we can proceed.